packages/calypso-e2e/src/lib/pages/login-page.ts

```diff
@@ -61,4 +81,51 @@ export class LoginPage {
 		// Make sure after logging in that everything stablizes, so we can continue with the next action!
 		await this.page.waitForLoadState( 'load' );
 	}
+
+	/**
+	 * Navigates to the Login page.
+	 */
+	async visit(): Promise< void > {
+		await this.page.goto( getCalypsoURL( 'log-in' ), { waitUntil: 'networkidle' } );
```

Comment:
Normally, the navigation part is handled in `LoginFlow` as part of the different login options, but I am leaning towards simplifying the structure of LoginPage and LoginFlow by merging the two.

It will be something handled in a follow-up PR, as this one is big enough already.